<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QWidget" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid rgb(100, 169,139);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="head" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>701</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#head{
background:rgb(0,188,212);
}</string>
      </property>
      <widget class="QToolButton" name="close_btn">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>2</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="min_btn">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>2</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QtMaterialTabs" name="navigate" native="true">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>0</y>
         <width>321</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#navigate{
background:rgb(255,0,0);
}</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mid" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>80</y>
        <width>701</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#mid{
background:rgb(234,242,249);
}</string>
      </property>
      <widget class="QtMaterialRaisedButton" name="search">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>10</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
      <widget class="ElaLineEdit" name="input_edit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>13</y>
         <width>391</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>请输入学工号/昵称</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bottom" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>701</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QStackedWidget" name="view_stack">
         <widget class="QWidget" name="user_page">
          <widget class="QListView" name="user_view">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>679</width>
             <height>289</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="group_page">
          <widget class="QListView" name="group_view">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>679</width>
             <height>289</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialTabs</class>
   <extends>QWidget</extends>
   <header location="global">qtmaterialtabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElaLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">elalineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
